Atomic/atomic.py

```diff
@@ -30,6 +41,22 @@
 from docker.errors import NotFound
 
 IMAGES = []
+ATOMIC_LIBEXEC = os.environ.get('ATOMIC_LIBEXEC', '/usr/libexec/atomic')
+
+OSTREE_DOCKERIMG_PREFIX = "dockerimg/"
+SYSTEMD_UNIT_FILES_DEST = "/etc/systemd/system"
```

Comment:
this is really a Docker layer/image, and it is the prefix of the branch which points to the docker image layer, these OSTree branches will look like:
dockerimg/CHECKSUM (for each layer)
dockerimg/image-name (for the image, which has information on what layers are used)